src/inc/quic_tls.h

```diff
@@ -134,8 +154,8 @@ typedef enum QUIC_TLS_RESULT_FLAGS {
 
 typedef enum QUIC_TLS_DATA_FLAGS {
 
-    QUIC_TLS_CRYPTO_DATA,
-    QUIC_TLS_TICKET_DATA
+    QUIC_TLS_CRYPTO_DATA                = 0x01,
+    QUIC_TLS_TICKET_DATA                = 0x02
```

Comment:
Why?

Reply:
Flags should start at 1 not at 0.

Reply:
I mean why does it need to be flags? Do you expect to somehow pass both types of data at the same time? If it's always going to be either/or then it doesn't need to be a flag.

Reply:
Good point. I thought we might want to have modifiers to the data in the future, so some flags would be exclusive, and others wouldn't. But we can make that change in the future, if we need to.